README.md

```diff
@@ -1,3 +1,117 @@
 # ownCloud Documentation UI
 
-TBD
+[link-Antora-Default-UI]: https://gitlab.com/antora/antora-ui-default
+[link-Antora]: https://antora.org
+[link-ui_bundle.zip]: https://minio.owncloud.com/documentation/ui-bundle.zip
+[link-README]: https://github.com/owncloud/docs
+[link-git]: https://git-scm.com
+[link-Node-8]: https://nodejs.org
+[link-Gulp-CLI]: http://gulpjs.com
+[link-Yarn]: https://yarnpkg.com
+[link-git-package]: https://git-scm.com/downloads
+[link-nvm]: https://github.com/creationix/nvm
+[link-nvm-installation-instructions]: https://github.com/creationix/nvm#installation
+
+The ``docs-ui`` repository is a custom version of the [Antora Default UI][link-Antora-Default-UI], 
+for the [Antora][link-Antora] version of the ownCloud documentation.
+ 
+## Use the ownCloud Antora-based Docs Default UI
+
+Any merged changes will automatically trigger the creation of a new ``ui-bundle.zip`` package file.
+This [ui_bundle.zip][link-ui_bundle.zip] file is automatically published and can be downloaded or used by scripts. 
+
+When compiling the documentation in the ``docs repro``, this ``ui-bundle.zip`` file is, if not manually 
+defined differently, automatically used.
+
+If you want to see the result of your unmerged UI changes made in a compiled documentation, 
+you need to manually  create a ``ui-bundle.zip`` and copy it to the root folder of your ``docs repro``. 
+This file will not be tracked. When compiling the documentation, you must use the ``--ui-bundle-url ui-bundle.zip`` 
+option in your ``antora``  command to use it instead the published version. 
+See the [README][link-README] in the ``docs reposirory`` for further information how to compile the documentation.
+
+## Package Creation for Use with Antora
+
+If you need to package the UI in order to preview unmerged changes in the UI in local development, 
+run the following command in the local root of your ``docs-ui`` clone:
+
+```
+ $ gulp pack
+```
+
+When the command has finished, the UI bundle will be available in directory ``build/ui-bundle.zip``
+
+NOTE: This file will not be used for production, see description above.
+
+## Prerequisites
+
+To create a local version of a ``ui-bundle.zip``, you need to install the following software on your computer.
```

Comment:
As it's not necessary to generate a ui-bundle archive, I suggest changing the wording to refer to users being able to run `gulp preview` instead.